Jenkinsfile

```diff
@@ -0,0 +1,2 @@
+// Build the plugin using https://github.com/jenkins-infra/pipeline-library
+buildPlugin(jenkinsVersions: [null, '2.107'])
```

Comment:
Why tests with 2.107 succeeds even with `java.level=7`?

Reply:
You do not _have_ to compile using Java 8 syntax just to run on 2.60+.